nucleus/src/main/java/za/co/absa/subatomic/application/team/TeamService.java

```diff
@@ -95,6 +96,36 @@ public String newDevOpsEnvironment(String teamId, String requestedBy) {
                 TimeUnit.SECONDS);
     }
 
+    public String updateMembershipRequest(String teamId,
+            MembershipRequestResource membershipRequest) {
+
```

Comment:
Please remove the blank lines after method declaration for consistency with the existing methods.